controllers/managedocs_controller.go

```diff
@@ -1786,3 +1795,30 @@ func (r *ManagedOCSReconciler) setDeviceSetCount(deviceSet *ocsv1.StorageDeviceS
 		deviceSet.Count = currDeviceSetCount
 	}
 }
+
+func (r *ManagedOCSReconciler) reconcileRookOverrideConfig() error {
+	if r.DeploymentType != consumerDeploymentType {
+		r.Log.Info("Non consumer deployment, skipping reconcile for rook override config")
+		return nil
+	}
+	r.Log.Info("Reconciling Rook Override Config Map")
+	_, err := ctrl.CreateOrUpdate(r.ctx, r.Client, r.rookConfigOverride, func() error {
+		if err := r.own(r.rookConfigOverride); err != nil {
+			return err
+		}
+		configMapData := make(map[string]string, 0)
+		globaldata := `
+		    [global]
+			osd_pool_pg_autoscale_mode = off
+			osd_pool_default_pg_num = 128
+			osd_pool_default_pgp_num = 128
+			`
+		configMapData["config"] = globaldata
```

Comment:
1. No need to use make
2. all of the extra spaces (indentations) might create issues

```suggestion
		r.rookConfigOverride.Data := map[string]string {
			"config": `[global]
osd_pool_pg_autoscale_mode = off
osd_pool_default_pg_num = 128
osd_pool_default_pgp_num = 128` ,
		}
```

Reply:
Done